src/plots/rectanglePlot.ts

```diff
@@ -231,6 +231,30 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Gets the Entities at a particular Point.
```

Comment:
But Entities is not. Plus it's not even returning an Entity, it's returning a PlotEntity.